mongodb_store/scripts/mongodb_play.py

```diff
@@ -156,9 +156,17 @@ def run(self, running):
         self.event.wait()
 
         timeout = 1
+        try:
+            action_goal_match = True if "Goal" in self.msg_type else False
+            msg_type_not_init = False
+        except AttributeError as e:
```

Comment:
sometimes self.msg_type is not initialized before this runs (async issue?). Not the best way to catch this issue, but it works fairly quickly. I would rather not do a substring match every single time we publish a topic